src/main/java/org/opensearch/agent/tools/SearchAnomalyDetectorsTool.java

```diff
@@ -82,7 +82,8 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
             ? Boolean.parseBoolean(parameters.get("highCardinality"))
             : null;
         final Long lastUpdateTime = parameters.containsKey("lastUpdateTime") && StringUtils.isNumeric(parameters.get("lastUpdateTime"))
-        ? Long.parseLong(parameters.get("lastUpdateTime")) : null;
+            ? Long.parseLong(parameters.get("lastUpdateTime"))
```

Comment:
add a test?

Reply:
this is a code style update to make spotless happy.